lib/active_snapshot/config.rb

```diff
@@ -1,31 +1,50 @@
+# frozen_string_literal: true
+
 module ActiveSnapshot
   class Config
+    class InvalidStorageMethodError < StandardError
+      def initialize(value)
+        super("Invalid storage_method provided: `#{value}`. Valid options are: #{STORAGE_METHODS.join(', ')}")
+      end
+    end
+
+    SERIALIZED_YAML = 'serialized_yaml'
+    SERIALIZED_JSON = 'serialized_json'
+    NATIVE_JSON = 'native_json'
```

Comment:
These constants would need to be frozen too.

Reply:
Already done via this like
https://github.com/westonganger/active_snapshot/pull/54/files/3c7e9664d479e9470ebfd6819d6d6c5fac685c11#diff-707ef8cdb9cc90fc88840313c058a0ea5efd86374ec17cd2d7a97b4a3157804dR1